ocp4/profiles/coreos-ncp.profile

```diff
@@ -77,6 +77,7 @@ selections:
     #- sshd_set_keepalive
     #- sshd_enable_warning_banner
     #- sshd_rekey_limit
+    
```

Comment:
nitpick: Unchaning change. But I'd rather not wait for Jenkins to test it again.